sdk/c/api/data_channel_interface.h

```diff
@@ -15,12 +15,43 @@
 
 extern "C" {
 #endif
-
 RTC_C_CLASS(webrtc::DataChannelInterface, WebrtcDataChannelInterface)
+RTC_C_CLASS(webrtc::DataChannelObserver, WebrtcDataChannelObserver)
+
+struct WebrtcDataChannelObserverFunctions {
+  void (*on_destruction)(void*);
+  void (*on_state_change)(void*);
+  void (*on_message)(void*, bool binary, const uint8_t* data, size_t len);
+  void (*on_buffered_amount_change)(void*, uint64_t);
+};
 
 RTC_EXPORT void webrtcDataChannelInterfaceRelease(
     const WebrtcDataChannelInterface* channel);
 
+RTC_EXPORT RtcString* webrtcDataChannelLabel(
+    const WebrtcDataChannelInterface* channel);
+
+RTC_EXPORT int webrtcDataChannelStatus(
```

Comment:
```suggestion
RTC_EXPORT int webrtcDataChannelState(
```

Follow the naming of the underlying function. It is crucial to make it easy to understand what feature it represents, especially because we do not provide additional documentations for the binding and users have to rely on the counterparts of the C++ APIs.

If you think such renaming is strongly preferred, it should be submitted to the upstream and reviewed by its maintainers.